config/prismjs/dvc.js

```diff
@@ -60,5 +60,82 @@ Prism.languages.dvc = {
       ...bash
     }
   },
+  table: {
```

Comment:
I think we should make it a separate syntax highlighter, what do you think to keep them simpler?

Reply:
When you say a separate syntax highlighter, are you referring to a highlighter separate from our `dvc` and `bash` highlighter?

Reply:
Yep 👍

Reply:
Wouldn't that force us to have to place all of our tables on their own line? Most of our tables are written as part of `dvc` code blocks 🤔

Reply:
yes, I think it's fine to put them separately. If feels like a pretty heave component with its own rules to try to squeeze it into `dvc` .

Reply:
If the docs devs want to be able to co-locate invocation and the resulting table, we can have the separate highlighter include both the regular `dvc` one as well as table logic, and name it something like `dvcexp`. That way, the convenient feature of highlighted command + output is available without bloating every single use of the `dvc` highlighter.

Reply:
Separate `dvc` and `dvctable` (or `dvcout`) syntaxes are fine. Separating command and output may allow finer control. (e.g. putting automated prompt `$` in front of `dvc`, but not `dvcout`.)

Reply:
I moved the table styles to a syntax highlighter called `dvctable`.

Reply:
> what do you think to keep them simpler?

What's made simpler by adding a highlighter instead of expanding on the existing one? Is there a benefit to separate them? E.g. it's easier to implement

---
Related but out-pf-scope Q: is it possible to expand on default ones like on `` ```shell-session ``?

Reply:
> Is there a benefit to separate them? E.g. it's easier to implement

As @rogermparent mentioned, one benefit is avoiding bloating every single use of our `dvc` highlighter, since only a small amount of our code blocks need table styling.

> Related but out-pf-scope Q: is it possible to expand on default ones like on ```shell-session (similar to bash)?

I'm pretty sure it's possible.